server/routes/offenceCodeDecisions/officerDecisionHelper.ts

```diff
@@ -16,14 +16,20 @@ enum ErrorType {
   OFFICER_MISSING_LAST_NAME_INPUT_SUBMIT = 'OFFICER_MISSING_LAST_NAME_INPUT_SUBMIT',
   OFFICER_MISSING_FIRST_NAME_INPUT_SEARCH = 'OFFICER_MISSING_FIRST_NAME_INPUT_SEARCH',
   OFFICER_MISSING_LAST_NAME_INPUT_SEARCH = 'OFFICER_MISSING_LAST_NAME_INPUT_SEARCH',
+  OFFICER_MISSING_ID_SUBMIT = 'OFFICER_MISSING_ID_SUBMIT',
 }
 const error: { [key in ErrorType]: FormError } = {
+  // We have separated out the submit and search validation messages here
   OFFICER_MISSING_FIRST_NAME_INPUT_SUBMIT: {
     href: '#officerSearchFirstNameInput',
-    text: 'Search for a prison officer',
+    text: 'Enter the person’s first name',
   },
   OFFICER_MISSING_LAST_NAME_INPUT_SUBMIT: {
     href: '#officerSearchLastNameInput',
+    text: 'Enter the person’s last name',
+  },
+  OFFICER_MISSING_ID_SUBMIT: {
```

Comment:
Minor one - the ordering here is different from `staffDecisionHelper.ts` which has `STAFF_MISSING_ID_SUBMIT` as the first entry